### DISCUSSION TITLE
Can you release a new version please?

### DISCUSSION THREAD

**Discussion by posva** [google/zx#748] [General] [answered]
Hello! I see there haven't been any releases for a while and I was wondering if you could release a new version soon. I was interested in https://github.com/google/zx/commit/b1ca2325f9a8e2c877badc486aaaf7fe686d7d76 since right now, one cannot use it in `npx zx https://example.com/my-script`.
Meanwhile the docs are showing this example with `minimist` that doesn't work

- https://google.github.io/zx/api#argv

```ts
const myCustomArgv = minimist(process.argv.slice(2), {
  boolean: [
    'force',
    'help',
  ],
  alias: {
    h: 'help',
  },
})
```

Sorry for the non-conforming issues 😓

**Comment by prunelsigma** [comment]
+1 I would love a new release to fix this, especially that it's already added in documentation https://google.github.io/zx/api#argv

**Comment by groie** [comment]
I was just as surprised as the others that this example in your documentation does not work. Could you please either fix the issue (preferable) or update the documentation. Thanks!

**Comment by ivancuric** [comment]
Gotta say it's a bit worrying that there hasn't been a release since Jul 6, 2023. Seems that the maintainer doesn't have enough time to work on this. I was hoping that google would support this a bit more, but I missed:

> This is not an officially supported Google product

**Comment by antonmedv** [comment]
Will release it)

**Reply by groie** [reply]
Hi Anton - any news?

**Comment by antongolub** [comment, marked as answer]
Finally,  https://github.com/google/zx/releases/tag/8.0.0
